public/app/core/components/AccessControl/PermissionListItem.tsx

```diff
@@ -12,42 +14,59 @@ interface Props {
   onChange: (item: ResourcePermission, permission: string) => void;
 }
 
-export const PermissionListItem = ({ item, permissionLevels, canSet, onRemove, onChange }: Props) => (
-  <tr>
-    <td>{getAvatar(item)}</td>
-    <td>{getDescription(item)}</td>
-    <td>{item.isInherited && <em className="muted no-wrap">Inherited from folder</em>}</td>
-    <td>
-      <Select
-        disabled={!canSet || !item.isManaged}
-        onChange={(p) => onChange(item, p.value!)}
-        value={permissionLevels.find((p) => p === item.permission)}
-        options={permissionLevels.map((p) => ({ value: p, label: p }))}
-      />
-    </td>
-    <td>
-      <Tooltip content={getPermissionInfo(item)}>
-        <Icon name="info-circle" />
-      </Tooltip>
-    </td>
-    <td>
-      {item.isManaged ? (
-        <Button
-          size="sm"
-          icon="times"
-          variant="destructive"
-          disabled={!canSet}
-          onClick={() => onRemove(item)}
-          aria-label={`Remove permission for ${getName(item)}`}
+export const PermissionListItem = ({ item, permissionLevels, canSet, onRemove, onChange }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <tr>
+      <td>{getAvatar(item)}</td>
+      <td>{getDescription(item)}</td>
+      <td>{item.isInherited && <em className="muted no-wrap">Inherited from folder</em>}</td>
+      <td>
+        <Select
+          disabled={!canSet || !item.isManaged}
+          onChange={(p) => onChange(item, p.value!)}
+          value={permissionLevels.find((p) => p === item.permission)}
+          options={permissionLevels.map((p) => ({ value: p, label: p }))}
         />
-      ) : (
-        <Tooltip content={item.isInherited ? 'Inherited Permission' : 'Provisioned Permission'}>
-          <Button size="sm" icon="lock" />
-        </Tooltip>
-      )}
-    </td>
-  </tr>
-);
+      </td>
+      <td>
+        {item.warning ? (
+          <Tooltip
+            content={
+              <>
+                <div className={styles.warningText}>{item.warning}</div>
```

Comment:
```suggestion
                <Box marginBottom={1}>{item.warning}</Box>
```

For simple classes like this you can just use [Box](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Layout/Box/Box.tsx)! 📦